test/e2e/network/networking.go

```diff
@@ -226,6 +226,36 @@ var _ = SIGDescribe("Networking", func() {
 			config.DialFromEndpointContainer("udp", config.NodeIP, config.NodeUDPPort, config.MaxTries, 0, config.EndpointHostnames())
 		})
 
+		ginkgo.It("should function for multiple endpoint-Services with same selector", func() {
+			config := e2enetwork.NewNetworkingTestConfig(f, false, false)
+			ginkgo.By("creating a second service with same selector")
+			svc2, udpPort := CreateSecondNodePortService(f, config)
+
+			// original service should work
+			ginkgo.By(fmt.Sprintf("dialing(udp) %v (endpoint) --> %v:%v (config.clusterIP)", config.EndpointPods[0].Name, config.ClusterIP, e2enetwork.ClusterUDPPort))
```

Comment:
Any reason to use udp instead of tcp here?

Reply:
No reason. I think I started with the UDP test which was right above. I will switch it to tcp.